Backend/GroceryStore_Backend/README.md

```diff
@@ -0,0 +1,27 @@
+
+## Docker File Creation
+
+dotnet publish -c release -o app/ .
+docker build --pull -t grocerystore_backend .
```

Comment:
I think the build should come first and then publish

Reply:
No, this is correct step . Please do check out this link  [https://medium.com/faun/deploy-dotnet-core-api-docker-container-with-mysql-on-heroku-ed387eab4222](url)